### PR TITLE
Lambda into terraform

### PR DESCRIPTION
**Pull request by Chufle**
I did some folder renamings and py file renaming. New folderstructure for lambda functions photo checkin. 
/photo-check/src/photo_checkin.py
Attention!!!: The py file is not doing the checkin of the photo to dynamo db. It's only a dummy-file which does some tests.
This pull request is only a intermediate step!

### CODE REVIEW

File: photo-checkin/requirements.tx [hunk 1]
```diff
@@ -0,0 +1 @@
+boto3==1.26.16
```
Comment (fabianschmauder) [inline photo-checkin/requirements.tx:1]: empty line

File: photo-checkin/src/photo_checkin.py [hunk 2]
```diff
@@ -1,14 +1,11 @@
...
 
 s3 = boto3.client('s3')
 
-def lambda_handler(event, context):
-    # print("Received event: " + json.dumps(event, indent=2))
-
+def handler(event, context):
     # Get the object from the event and show its content type
     bucket = event['Records'][0]['s3']['bucket']['name']
     key = urllib.parse.unquote_plus(event['Records'][0]['s3']['object']['key'], encoding='utf-8')
```
Comment (fabianschmauder) [inline photo-checkin/src/photo_checkin.py:11]: you could get multiple records. therefore use a for loop

**Review by fabianschmauder — COMMENTED** [review]
just small stuff

File: photo-checkin/src/photo_checkin.py (hunk 2 above)
Comment (Chufle) [reply photo-checkin/src/photo_checkin.py:11]: ok

**Pull request merged**